incubator/zalenium/values.yaml

```diff
@@ -0,0 +1,199 @@
+# Default values for zalenium.
```

Comment:
should have a way override this values, exp:
```
$ envsubst override.tpl.yaml > override.yaml
$ $ helm upgrade --install --name example-name teracy-incubator/zalenium -f values.yaml  -f override.yaml
```

Reply:
should use our current styles, sent from Slack

Reply:
We don't use current styles! It will be built as a package.